local-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+pushd play-java-chatroom-example        && scripts/test-sbt && popd
+pushd play-java-compile-di-example      && scripts/test-sbt && popd
+pushd play-java-dagger2-example         && scripts/test-sbt && popd
+pushd play-java-ebean-example           && scripts/test-sbt && popd
+pushd play-java-fileupload-example      && scripts/test-sbt && popd
+pushd play-java-forms-example           && scripts/test-sbt && popd
+pushd play-java-grpc-example            && scripts/test-sbt && popd
+pushd play-java-hello-world-tutorial    && scripts/test-sbt && popd
+pushd play-java-jpa-example             && scripts/test-sbt && popd
+pushd play-java-rest-api-example        && scripts/test-sbt && popd
+pushd play-java-starter-example         && scripts/test-sbt && popd
+pushd play-java-streaming-example       && scripts/test-sbt && popd
+pushd play-java-websocket-example       && scripts/test-sbt && popd
+pushd play-java-akka-cluster-example    && scripts/test-sbt && popd
+pushd play-scala-anorm-example          && scripts/test-sbt && popd
+pushd play-scala-chatroom-example       && scripts/test-sbt && popd
+pushd play-scala-compile-di-example     && scripts/test-sbt && popd
+pushd play-scala-fileupload-example     && scripts/test-sbt && popd
+pushd play-scala-forms-example          && scripts/test-sbt && popd
+pushd play-scala-grpc-example           && scripts/test-sbt && popd
+pushd play-scala-hello-world-tutorial   && scripts/test-sbt && popd
+pushd play-scala-isolated-slick-example && scripts/test-sbt && popd
+pushd play-scala-log4j2-example         && scripts/test-sbt && popd
+pushd play-scala-macwire-di-example     && scripts/test-sbt && popd
+pushd play-scala-rest-api-example       && scripts/test-sbt && popd
+# Uses libsodium
+# pushd play-scala-secure-session-example && scripts/test-sbt && popd
+pushd play-scala-slick-example          && scripts/test-sbt && popd
+pushd play-scala-starter-example        && scripts/test-sbt && popd
+pushd play-scala-streaming-example      && scripts/test-sbt && popd
+# uses vanilla sbt
+# pushd play-scala-tls-example            && scripts/test-sbt && popd
+pushd play-scala-websocket-example      && scripts/test-sbt && popd
```

Comment:
I'm sneaking this one in as an early attempt to quickly run samples locally for faster diagnosis in case of regression.

Reply:
We could add something like https://github.com/akka/akka-platform-guide/blob/main/scripts/format-all.sh and even remove the separate Travis jobs.
WDYT?

Reply:
> We could add something like [https://github.com/akka/akka-platform-guide/blob/main/scripts/format-all.sh](https://github.com/akka/akka-platform-guide/blob/main/scripts/format-all.sh?rgh-link-date=2020-11-05T12%3A13%3A36Z) and even remove the separate Travis jobs.
> WDYT?

Hmm, yeah, simplifying this code is a good idea.

I'm not sure about running all tests in a single travis job. Every once in a while there's a failure here or there and having separate jobs makes it faster to diagnose. Let's merge this as is and discuss improvements in a new issue.

Reply:
raised #118